nightfall-client/src/services/commitment-storage.mjs

```diff
@@ -753,14 +755,61 @@ export async function findUsableCommitmentsMutex(
   );
 }
 
+/**
+ *
+ * @function saveCommitments save a list of commitments in the database
+ * @param {[]} listOfCommitments a list of commitments to be saved in the database
+ * @returns if all the commitments in the list already exists in the database
+ * throw an error, else return a success message.
```

Comment:
You can use the `@throws {Error}` to explain the "throw error" bit

Reply:
Added.